Baikal/PostEffects/ML/denoiser.h

```diff
@@ -16,10 +46,30 @@ namespace Baikal
             kColorDepthNormalGloss7,
         };
 
-        std::unique_ptr<Inference> CreateMLDenoiser(MLDenoiserInputs inputs,
-                                                    float gpu_memory_fraction,
-                                                    std::string const& visible_devices,
-                                                    std::size_t width,
-                                                    std::size_t height);
+        class MLDenoiser : public PostEffect
+        {
+        public:
+
+            // Constructor
+            MLDenoiser(CLWContext context, Inference::Ptr inference, MLDenoiserInputs inputs);
```

Comment:
Forward declaration is not working here. To make it working, use CLWContext const&.